### ISSUE TITLE
Any plans to add this to other browsers?

### ISSUE THREAD

**Issue by JIGASHI** [Sleighs/chat-plus#1]
Firefox etc.

**Comment by Sleighs** [comment]
Yeah. Putting together a new update and making it available on Firefox.

**Comment by Sleighs** [comment]
The extension is available on Firefox, as of today.

https://addons.mozilla.org/en-US/firefox/addon/chatplus-for-rumble/
